### PR TITLE
[combined] fix typo 

### PR DESCRIPTION
**Pull request by xulike666**
Signed-off-by: Aaron.L.Xu <likexu@harmonycloud.cn>

**- What I did**

[Newbie Task : 6/36]

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
@xulike666 would it be possible to combine your fixes to a single PR?

**Comment by xulike666** [comment]
@thaJeztah , thanks for your suggestion.  My subjective willingness is to combine those boring  bothersome typo fixes for sure but my superiors who are so care about the number of commits for docker community which is 36 in my case. Apologize for next 30 typo-fix pr and from a different perspective，I'm try to sketch the whole project structure for my No.37 worthwhile contribution.

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Comment by runcom** [comment]
LGTM

**Comment by AkihiroSuda** [comment]
But please squash next 30 planned typo fixes? 😅

**Comment by xulike666** [comment]
Well gentlemen, I have already hack a fix-typo-pr-bot for this kind of daily PRs and won't put the next 'episode' out in this LGTM.

**Comment by thaJeztah** [comment]
@xulike666 "number of commits" should not be a metric. Although we appreciate your contributions, also keep in mind that each PR will at least result in 6 machines (three on the PR, three or more on "master") running CI for an hour each

**Comment by xulike666** [comment]
@thaJeztah, nice guider !  As a freshman, I'd like to squash my typo-related contributions into one pr ASAP and apologize for wasting computing resources.

**Comment by thaJeztah** [comment]
> I'd like to squash my typo-related contributions into one pr 

That'd be appreciated

> apologize for wasting computing resources

Well, they're there to be running, but could delay other PR's from getting merged. Overall "measuring performance" just by looking at number of commits is _not_ the way (not your fault, but something for your superiors to rethink); Contributing is _more_ than just generating commits; looking into bugs, trying to reproduce those, or assisting others are great way to contribute, just less easy to "measure", but not less appreciated.

**Comment by xulike666** [comment]
@AkihiroSuda, sorry for the delay in this pr and I rebase and push all my typo errors  I found  in these days .

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
Please omit unrelated commits

File: pkg/term/term_windows.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ const (
 // vtInputSupported is true if enableVirtualTerminalInput is supported by the console
 var vtInputSupported bool
 
-// StdStreams returns the standard streams (stdin, stdout, stedrr).
+// StdStreams returns the standard streams (stdin, stdout, stderr).
```
Comment (allencloud) [inline pkg/term/term_windows.go]: I also found that in file `pkg/term/term.go#L34` there is a same typo.
Could you please combine that in this single PR?

File: integration-cli/docker_api_exec_test.go [hunk 2]
```diff
@@ -65,7 +65,7 @@ func (s *DockerSuite) TestExecAPICreateContainerPaused(c *check.C) {
 }
 
 func (s *DockerSuite) TestExecAPIStart(c *check.C) {
-	testRequires(c, DaemonIsLinux) // Uses pause/unpause but bits may be salvagable to Windows to Windows CI
+	testRequires(c, DaemonIsLinux) // Uses pause/unpause but bits may be salvageable to Windows to Windows CI
```
Comment (allencloud) [inline integration-cli/docker_api_exec_test.go]: also found in `pkg/archive/copy_unix_test.go#L3`

**Comment by AkihiroSuda** [comment]
And please sign your commit

**Comment by allencloud** [comment]
That is great. Thanks for your work. While I think it would be better if you could help to squash the 3 commits into a single one. @xulike666

**Review by AkihiroSuda — APPROVED** [review]
LGTM if green

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**